code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -328,7 +328,7 @@ var/global/list/damage_icon_parts = list()
 	overlays += O
 	obj_overlays[HAIR_LAYER] = O
 
-	if(update_icons)   update_icons()
+/mob/living/carbon/human/proc/update_beard(var/update_icons=1)
```

Comment:
I'm trying to find things that would cause the suit storage thing,  but I got nothing.